frontend/src/components/security/signup/Signup.tsx

```diff
@@ -35,11 +36,11 @@ function Signup(props: Props) {
     }
     const confirmPassword = (e: ChangeEvent<HTMLInputElement>, repeat: boolean) => {
         const password = repeat ? credentials.password : repeatedPassword;
-        setPasswordConfirmed(password.length > 3 && password === e.target.value) ;
+        setPasswordConfirmed(password.length > 3 && password === e.target.value);
```

Comment:
3 is a magic number and maybe its another feature to create a reasonable password-strength-check at signup